static/bidder-info/360playvid.yaml

```diff
@@ -0,0 +1,22 @@
+aliasOf: "teqblaze"
+endpoint: "https://ssp.360playvid.com/pserver"
+maintainer:
+  email: "prebid@360playvid.com"
+capabilities:
+  site:
+    mediaTypes:
+      - banner
+      - video
+      - native
+  app:
+    mediaTypes:
+      - banner
+      - video
+      - native
+userSync:
+  redirect:
+    url: "https://cookie.360playvid.com/pbserver?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&ccpa={{.USPrivacy}}&gpp={{.GPP}}&gpp_sid={{.GPPSID}}&redirect={{.RedirectURL}}"
+    userMacro: "[UID]"
+  iframe:
+    url: "https://cookie.360playvid.com/pbserverIframe?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&ccpa={{.USPrivacy}}&gpp={{.GPP}}&gpp_sid={{.GPPSID}}&pbserverUrl={{.RedirectURL}}"
+    userMacro: "[UID]"
```

Comment:
Verified iframe works with a request being made to PBS setuid with the 360playvid uid in the PBS cookie
<img width="1258" height="102" alt="Screenshot 2025-08-28 at 2 42 20 PM" src="https://github.com/user-attachments/assets/e6e344ce-e911-420a-b444-2822b4b926f4" />